pkg/skaffold/runner/runner.go

```diff
@@ -271,12 +271,12 @@ func (r *SkaffoldRunner) Dev(ctx context.Context, out io.Writer, artifacts []*la
 			return ErrorConfigurationChanged
 		case len(changed.needsResync) > 0:
 			for _, s := range changed.needsResync {
-				if err := r.Syncer.Sync(s); err != nil {
+				color.Default.Fprintf(out, "Syncing %d files for %s\n", len(s.Copy)+len(s.Delete), s.Image)
```

Comment:
update to latest code

Reply:
Yeah just maybe the last commit. I changed this message to the debug and info statement.

Reply:
Oh, the one I asked you to add? It didn't work well in reality :-) Didn't give enough feedback

Reply:
:) SGTM